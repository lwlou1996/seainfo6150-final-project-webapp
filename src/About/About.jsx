import React from 'react'
import NaviBar from "../Bar/NaviBar/NaviBar";
import AboutImg from "../images/mainpage.png"

import styles from "./About.module.css";
import {Link} from "react-router-dom";

const About = () => {

    return(
        <div className={styles.container}>
            <div>
                <NaviBar />
            </div>

            <div className={styles.imgContainer}>
                <img className={styles.errorImg} src={AboutImg} alt="About Image: just to be simple"/>
            </div>

            <div className={styles.aboutText}>
                <h2><a href="https://www.linkedin.com/in/leichenzhou/">ABOUT ME</a></h2>

                <ul>
                    <li>Understand basic usability, user experience and accessibility principles</li>
                    <li>Have a basic understanding and practical knowledge of HTML5</li>
                    <li>Write valid, well-formed, scalable, and semantically appropriate HTML5 within the context of a React app</li>
                    <li>Have a basic understanding and practical knowledge of CSS</li>
                    <li>Write CSS stylesheets within the context of a React app</li>
                    <li>Have a basic understanding and practical knowledge of Javascript</li>
                    <li>Have a basic understanding and practical knowledge of React and app development within the context of a React app.</li>
                    <li>Identify the types of images used in modern web design and explain what types are appropriate for different functions </li>
                </ul>
            </div>

        </div>
    );

}

export default About;