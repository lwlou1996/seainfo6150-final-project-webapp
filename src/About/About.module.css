.container {
    display: flex;
    flex-flow: column wrap;
    /*justify-content: center;*/

    text-align: center;
}

.imgContainer {
    /*text-align: center;*/
}

.errorImg {
    width: 50%;

    padding-top: 50px;
}

.aboutText {
    margin: 100px auto;
    text-align: left;
}

.aboutText a {
    text-decoration: underline;
}