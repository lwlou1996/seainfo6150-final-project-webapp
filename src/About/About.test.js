import React from 'react'
import About from './About'
import {render} from "@testing-library/react";

describe("About test", () => {

    it('about render correctly ', () => {
        const { container } = render(<About />);

        expect(container).toMatchSnapshot();
    });

});