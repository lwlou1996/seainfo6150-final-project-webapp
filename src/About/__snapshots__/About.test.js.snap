// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About test about render correctly  1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <div
        class="navContainer"
      >
        <header
          class="container"
        >
          <nav>
            <a
              class="mainIcon"
              href="/"
            >
              PoSub
            </a>
          </nav>
          <nav>
            <ul
              class="list"
            >
              <li
                class="listItem"
              >
                <a
                  href="/contact"
                >
                  Contact
                </a>
              </li>
              <li
                class="listItem"
              >
                <a
                  href="/category"
                >
                  Category
                </a>
              </li>
            </ul>
          </nav>
        </header>
      </div>
    </div>
    <div
      class="imgContainer"
    >
      <img
        alt="About Image: just to be simple"
        class="errorImg"
        src="mainpage.png"
      />
    </div>
    <div
      class="aboutText"
    >
      <h2>
        <a
          href="https://www.linkedin.com/in/leichenzhou/"
        >
          ABOUT ME
        </a>
      </h2>
      <ul>
        <li>
          Understand basic usability, user experience and accessibility principles
        </li>
        <li>
          Have a basic understanding and practical knowledge of HTML5
        </li>
        <li>
          Write valid, well-formed, scalable, and semantically appropriate HTML5 within the context of a React app
        </li>
        <li>
          Have a basic understanding and practical knowledge of CSS
        </li>
        <li>
          Write CSS stylesheets within the context of a React app
        </li>
        <li>
          Have a basic understanding and practical knowledge of Javascript
        </li>
        <li>
          Have a basic understanding and practical knowledge of React and app development within the context of a React app.
        </li>
        <li>
          Identify the types of images used in modern web design and explain what types are appropriate for different functions 
        </li>
      </ul>
    </div>
  </div>
</div>
`;
